Userland/Libraries/LibJS/Runtime/TypedArrayPrototype.cpp

```diff
@@ -1715,6 +1716,69 @@ JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::to_spliced)
     return return_array;
 }
 
+// 1.2.2.1.6 %TypedArray%.prototype.with ( index, value ), https://tc39.es/proposal-change-array-by-copy/#sec-%typedarray%.prototype.with
+JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::with)
+{
+    auto index = vm.argument(0);
```

Comment:
Please also do `auto value = ...` here instead of accessing it by index below :)